src/pages/buy-sell/screens/BuySellTable/BuySellTable.tsx

```diff
@@ -32,14 +48,14 @@ const BuySellTable = () => {
     } = api.advert.useGetList({
         advertiser_name: searchValue,
         counterparty_type: activeTab === ADVERT_TYPE.BUY ? BUY_SELL.BUY : BUY_SELL.SELL,
-        local_currency: selectedCurrency,
+        local_currency: filteredCurrency,
         payment_method: selectedPaymentMethods.length > 0 ? selectedPaymentMethods : undefined,
-        sort_by: sortDropdownValue,
+        sort_by: sortByValue,
         use_client_limits: shouldUseClientLimits ? 1 : 0,
     });
 
     const onToggle = (value: string) => {
```

Comment:
```suggestion
    const onToggle = (value: TSortByValues) => {
```
wouldn't this be more type safe?

Reply:
we could but because the RadioGroupFilterModal is used in two different places, i made it more generic to just accept string, and ill just coerce here